packages/session-replay-browser/package.json

```diff
@@ -28,6 +28,7 @@
     "lint": "yarn lint:eslint & yarn lint:prettier",
     "lint:eslint": "eslint '{src,test}/**/*.ts'",
     "lint:prettier": "prettier --check \"{src,test}/**/*.ts\"",
+    "postinstall": "rimraf ../../node_modules/@medv/finder/finder.ts # This is required until the package is fixed upstream",
```

Comment:
It seems like we also having error when installing the lastest reply plugin via npm for throwing rimraf not found error.

Reply:
Thanks for letting me know. Let me take a look.

Reply:
I just published a change for:
```
Successfully published:
 - @amplitude/plugin-session-replay-browser@1.6.1
 - @amplitude/session-replay-browser@1.9.1
```
Can you check to see if these fix your issue? If not, would you mind opening an issue with more details?

Reply:
yes, it works now :) Thanks for the quick update!